### PR TITLE
remove necessity for license for oneAPI versions (>= 2021.x) in itac easyblock

### PR DESCRIPTION
**Pull request by robert-mijakovic**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyblocks/i/itac.py [hunk 1]
```diff
@@ -94,7 +102,10 @@ def install_step(self):
             except OSError as err:
                 raise EasyBuildError("Directory %s can't be created: %s", tmpdir, err)
 
-            cmd = "./install.sh --tmp-dir=%s --silent=%s" % (tmpdir, silentcfg)
+            if LooseVersion(self.version) >= LooseVersion('2021'):
+                cmd = "./install.sh -a -s --eula accept --install-dir=%s" % self.installdir
```
Comment (Micket) [inline easybuild/easyblocks/i/itac.py:106]: We should tie this acceptance to `eb --accept-eula-for=itac` instead of hardcoding accept.

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyblocks/i/itac.py (hunk 1 above)
Comment (boegel) [reply easybuild/easyblocks/i/itac.py:106]: +1 on that

Comment (robert-mijakovic) [reply easybuild/easyblocks/i/itac.py:106]: +1

Comment (robert-mijakovic) [reply easybuild/easyblocks/i/itac.py:106]: Resolved!

**Comment by boegel** [comment]
Test report by @boegel
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _itac-2017.1.024.eb_ 
 * **SUCCESS** _itac-2018.1.017.eb_ 
 * **SUCCESS** _itac-2018.3.022.eb_ 
 * **SUCCESS** _itac-2019.2.026.eb_ 
 * **SUCCESS** _itac-2019.4.036.eb_ 
 * **SUCCESS** _itac-8.0.0.011.eb_ 
 * **SUCCESS** _itac-8.1.4.045.eb_ 
 * **SUCCESS** _itac-9.0.3.051.eb_ 

Build succeeded for 8 out of 8 (8 easyconfigs in total)
node2613.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/fbdb51bd8401c29cde9f54281516de2a for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Pull request merged**